packages/utils/src/subgraphHealth.ts

```diff
@@ -1,6 +1,18 @@
 import axios from "axios";
 
-const SUBGRAPH_HEALTH_URL = "https://api.thegraph.com/index-node/graphql";
+// TODO get from chainData
+const GET_SUBGRAPH_HEALTH_URL = (name: string): string | undefined => {
```

Comment:
nit: change the input param from `name` to `url` bc we actually input the url

The `name` value is something like "nxtp-bsc-v1-runtime"